schema/core.md

```diff
@@ -544,19 +544,24 @@ Multiple locations can be specific using a list: (from [SOSO](https://github.com
 ### Associated Media
 
 **AssocatedMedia** is a property of `CreativeWork` for describing media objects 
-that encode the work. For example, this may be a text file that represents the
-data that is being cataloged. 
+that encode the work. Specific types of media objects that we selected for the I-GUIDE 
+data catalog are `DataDownload`, `ImageObject`, and `VideoObject`. Note that a media object 
+could have several properties from `CreativeWork`, but most importantly, it requires 
+`contentUrl`, `encodingFormat`. `contentUrl` is actual bytes of the media object and it is 
```

Comment:
it requires `contentUrl` and `encodingFormat`. `contentUrl` should point at the actual bytes of the media object and is represented as a URL.

Reply:
The sentence was revised. See commit 2142d6a.